src/config/index.js

```diff
@@ -8,6 +8,7 @@ let config = {
   storybookEnabled: false,
 }
 
+const __DEV__ = false
 if (__DEV__) {
```

Comment:
Since `__DEV__` doesn't seem to be working with create-react-app anymore, we should be able to accomplish the dev environment check like so:

```suggestion
const isDev = process.env.NODE_ENV !== 'production'
if (isDev) {
```